Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -5180,6 +5180,21 @@ PluginsEnabled:
     WebCore:
       default: false
 
+PointerLockOptionsEnabled:
+  type: bool
+  status: preview
```

Comment:
I think we'd need to have a resolved standards position prior to enabling in preview.

Reply:
Will move to `testable` then.

Reply:
Sounds good!